data_library/serial_interval/parse_data_si.py

```diff
@@ -45,7 +45,7 @@
 
 def write_ser_int_data(name):
     """Write a new csv file for 60-day-long serial intervals
-    to be used for the analysis of the Australian data.
+    to be used for the analysis of the COVID-19 data.
 
```

Comment:
In one of the docstrings, can you add a sentence saying that we use 1000 randomly selected samples of the lognormal parameters.

Also, a few lines below the comment "# Select data from the given state" is probably from an old file, I would update it to say from the parameters file.